### PR TITLE
Update compatibility table for Iron

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,13 @@ Galactic | Edifice | [galactic](https://github.com/gazebosim/ros_gz/tree/galacti
...
 Humble | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
 Humble | Garden | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | only from source
+Iron | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
+Iron | Garden | [humble](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
 Rolling | Edifice | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
-Rolling | Fortress | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | https://packages.ros.org
+Rolling | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
 Rolling | Garden | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
 
+
```
Comment (ahcorde) [inline README.md:17]: remove this new line

File: README.md [hunk 2]
```diff
@@ -8,10 +8,13 @@ Galactic | Edifice | [galactic](https://github.com/gazebosim/ros_gz/tree/galacti
 Galactic | Fortress | [galactic](https://github.com/gazebosim/ros_gz/tree/galactic) | only from source
 Humble | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
 Humble | Garden | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | only from source
+Iron | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
+Iron | Garden | [humble](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
 Rolling | Edifice | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | only from source
-Rolling | Fortress | [ros2](https://github.com/gazebosim/ros_gz/tree/ros2) | https://packages.ros.org
+Rolling | Fortress | [humble](https://github.com/gazebosim/ros_gz/tree/humble) | https://packages.ros.org
```
Comment (ahcorde) [inline README.md:14]: why this change?

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mjcarroll) [reply README.md:14]: This actually reflects what is being released into rolling: https://github.com/ros2-gbp/ros_ign-release#ros_gz-rolling---024410-1

**Review by ahcorde — APPROVED** [review]
LGTM without the new line
